src/library/scala/collection/mutable/OpenHashMap.scala

```diff
@@ -266,11 +266,6 @@ class OpenHashMap[Key, Value](initialSize : Int)
     table.foreach(entry => if (entry != null && entry.value != None) f(entry))
   }
 
-  def transform(f : (Key, Value) => Value): this.type = {
```

Comment:
Since `transform` is deprecated it didn't seem to make sense to implement this here. I could override `mapInPlace` but because of the way `mapInPlace` works we may need to add new keys as well, so the implementation is a bit more complex.

Reply:
OK. I think we will eventually replace `mapInPlace` with `mapValuesInPlace`, which is equivalent to `transform`. In such a case it will make sense to have it overridden in `OpenHashMap`. But we can do all that in another PR.